cli/README.md

```diff
@@ -55,11 +53,11 @@ func main() {
 
 ### Flags
 
-Flag, different from `pflag.Flag`, is an interface containing the following methods in Mamba.
+Flag, different from `pflag.Flag`, is an interface containing the following methods in CLI.
 
-And it is declarative in Mamba, unlike in Cobra where it is imperative which makes the flags **more readable**.
+And it is declarative in CLI, unlike in Cobra where it is imperative, which makes the flags **more readable**.
 
-Mamba will bind `viper` and `flags` automatically. That means you can get all your flag values from `viper`.
+CLI will bind `viper` and `flags` automatically. That means you can get all your flag values from `viper`.
```

Comment:
Do you still need to mention `viper` here?  If so, please link and explain what it is.  (I'd prefer we drop the new concept for users).